README.md

```diff
@@ -1 +1,139 @@
+<a name="readme-top"></a>
+
+# COMPUTER SCIENCE PEDAGOGIC SEMINER <a name="about-project"></a>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+
+# 📖 Computer Science Pedagogic Seminer <a name="about-project"></a>
+
+
+[**Computer Science Pedagogic Seminer**](https://tatouroberto.github.io/computer_seminer.github.io/) is a website for annual computer serminars.
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+
+<summary>Front-End Tech</summary>
+  <ul>
+    <li><a href="https://github.com/">Github</a></li>
+    <li><a href="https://html.com/">Html</a></li>
+     <li><a href="https://developer.mozilla.org/es/docs/Web/CSS">Css</a></li>
+    <li><a href="https://www.javascript.com/">Javascript</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **GitHub-Workflow**
+- **responsive**
+- **Dynamic**
+- **Maintainable**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+In order to run this project you need:
+- Node JS
+- Any code editor
+- Any browser
+
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+git https://github.com/tatouroberto/captone1_project
+```
+
+### Install
+
+Install this project with:
+
+
+```sh
+npm install
+```
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+
+
+👤 **TATOU ROBERTO**
+
+
+- GitHub: [@tatouroberto](https://github.com/tatouroberto)
+- Twitter: [@Tatou_rob](https://twitter.com/Tatou_rob)
+- LinkedIn: [tatou-roberto-496b1413b](https://www.linkedin.com/in/tatou-roberto-496b1413b/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **Add the commenting functionality**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/tatouroberto/captone1_project/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project you can download it and give it a star⭐️.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I will like to thank Microverse for giving me the opportunity to become a professional developer
```

Comment:
- [ ] The README should include an "Acknowledgements" section or a similar segment that provides proper credit to [Cindy Shin](https://www.behance.net/adagio07), who is the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). This credit is essential to adhere to the requirements of the [Creative Commons License](https://creativecommons.org/licenses/). Kindly ensure that you add this section to your README as part of acknowledging the original design and complying with licensing obligations. Your attention to this detail will contribute to the transparency and legality of your project.